ee/server/service/devices.go

```diff
@@ -184,20 +182,15 @@ func (svc *Service) TriggerLinuxKeyEscrow(ctx context.Context, host *fleet.Host)
 	if err != nil {
 		return err
 	}
-	if !ac.MDM.EnabledAndConfigured {
-		return fleet.ErrMDMNotConfigured
-	}
 
 	// Is AppConfig.MDM.EnableDiskEncryption the disk encryption setting for “No team”, while
 	// TeamMDM.EnableDiskEncryption is for all other teams?
 	// confirm No team taemId
-	if host.TeamID == nil {
-		// No team
+	if host.TeamID == nil { // No team
 		if !ac.MDM.EnableDiskEncryption.Value {
-			return &fleet.BadRequestError{Message: "Disk encryption is not enabled for this host"}
+			return &fleet.BadRequestError{Message: "Disk encryption is not enabled for this host's team"}
```

Comment:
@iansltx I just remembered why I put the message as I did initially. When a host is on "No team," it doesn't exactly make sense to refer to its team. I think "...not enabled for this host" makes a bit more sense in that regard, do you agree?